pkg/types/external/doc.go

```diff
@@ -0,0 +1,6 @@
+// Package none contains generic structures for installer
+// configuration and management.
+package external
+
+// Name is name for the None platform.
```

Comment:
should this read:
```suggestion
// Name is name for the External platform.
```
?

Reply:
Yep, it should.  I was troubleshooting the unit tests yesterday afternoon and am picking that back up today.